sample/src/main/java/com/airbnb/paris/sample/SectionView.kt

```diff
@@ -93,7 +93,9 @@ class SectionView : FrameLayout {
         private val DEFAULT_TITLE_STYLE = textViewStyle {
             add(TEXT_HEADLINE)
             backgroundRes(android.R.color.transparent)
-            letterSpacing(0f)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
@shaishavgandhi05 why is this necessary? `TextViewProxy` already specifies `@RequiresApi(Build.VERSION_CODES.LOLLIPOP)` for this attribute so the generate code should already have a similar check.